docs/source/en/api/workflows.md

```diff
@@ -0,0 +1,217 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Shareable workflows
+
+<Tip warning={true}>
+
+Workflows are experimental at the moment 🧪 Its APIs can change in future.
+
+</Tip>
+
+Workflows provide a simple mechanism to share your pipeline call arguments with others. It makes reproducibility of results easier. This doc shows you how you can leverage workflows for different use cases. 
+
+## Serializing a workflow
+
+```python
+from diffusers import DiffusionPipeline
+import torch
+
+pipeline = DiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16, safety_checker=None)
+pipeline.to("cuda")
+
+outputs = pipeline("A painting of a horse", num_inference_steps=15, return_workflow=True)
+workflow = outputs.workflow
+```
+
+As you can notice, by specifying `return_workflow=True` in the pipeline call, you can obtain a workflow that looks like so:
+
+```bash
+{'prompt': 'A painting of a horse',
+ 'height': 512,
+ 'width': 512,
+ 'num_inference_steps': 15,
+ 'guidance_scale': 7.5,
+ 'negative_prompt': None,
+ 'num_images_per_prompt': 1,
+ 'eta': 0.0,
+ 'output_type': 'pil',
+ 'return_dict': True,
+ 'callback': None,
+ 'callback_steps': 1,
+ 'cross_attention_kwargs': None,
+ 'guidance_rescale': 0.0,
+ 'clip_skip': None,
+ '_name_or_path': 'runwayml/stable-diffusion-v1-5'}
+```
+
+`workflow` is a [`Workflow`] object. It provides the values of all the arguments present in the `__call__()` of a pipeline.
+
+Once you have generated a workflow object, you can serialize it like so:
+
+```python
+outputs.workflow.save_workflow("my-simple-workflow-sd")
+```
+
+By default, your workflows will be saved with the following name: `diffusion_workflow.json`. But you can customize it by specifying the `filename` argument:
+
+```python
+outputs.workflow.save_workflow("my-simple-workflow-sd", filename="my_workflow.json")
+```
+
+By specifying `push_to_hub=True` in [`Workflow.save_workflow`], you can directly push the workflow object to the Hub. 
+
+## Loading a workflow
+
+You can load a workflow in a pipeline like so:
+
+```python
+from diffusers import DiffusionPipeline
+import torch
+
+pipeline = DiffusionPipeline.from_pretrained(
+    "runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16
+).to("cuda")
+
+pipeline.load_workflow("sayakpaul/my-simple-workflow-sd")
+```
+
+Once the pipeline is loaded with the desired workflow, it's ready to be called:
+
```

Comment:
From the Known limitation section, this might be more valuable to know up here instead of at the end.
```suggestion
<Tip>

You could also pass `prompt_embeds` instead of a `prompt`.

</Tip>
```